examples/client.c

```diff
@@ -307,6 +307,22 @@ check_token(coap_pdu_t *received) {
     memcmp(received->token, the_token.s, the_token.length) == 0;
 }
 
+static int
+event_handler(coap_context_t *ctx,
```

Comment:
ctx and session need the UNUSED_PARAM attribute to silence the compiler warning.

Reply:
Fixed - No idea how I missed the compiler warning, but ....